pkg/workloads/cortex/client/cortex/client.py

```diff
@@ -29,11 +30,17 @@
 
 
 class Client(object):
-    def __init__(
-        self, aws_access_key_id, aws_secret_access_key, operator_url, workspace="./workspace"
-    ):
+    def __init__(self, aws_access_key_id, aws_secret_access_key, operator_url):
+        """Initialize a Client to a Cortex Operator
+        
+        Args:
+            aws_access_key_id (string): AWS access key associated with the account that created the Cortex Cluster
+            aws_secret_access_key (string): AWS secrey key associated with the provided AWS access key  
+            operator_url (string): Operator URL of your cortex cluster
+        """
+
         self.operator_url = operator_url
-        self.workspace = workspace
+        self.workspace = str(Path.home() / ".cortex" / "workspace")
```

Comment:
nit: Is there something like `path.Join()` that would be safer to use here? In either case, `".cortex" / "workspace"` can probably be `".cortex/workspace"` instead